keyboards/chew/split/keyboard.json

```
@@ -19,7 +8,6 @@
             ["GP7", "GP6", "GP5", "NO_PIN", "NO_PIN"]
         ]
     },
-    "processor": "RP2040",
     "split": {
         "enabled": true,
         "matrix_pins": {
```

The previous submission used EE_HANDS, is there now a new PCB revision that supports handedness by pin?

```suggestion
        "handedness": {
            "pin": "GP10"
        },
        "matrix_pins": {
```

---

Yes, gp10 on the left and gp12 on the right are on +vcc.
Thanks to that I can use these pins as a kind of vbus.
(There were the last free pins thanks to the uart half-duplex)

---

Humm, now I have some doubts about that.
We could eventually remove this option to leave the user free to decide in the keymap config.
In this case I will be able to use the full duplex with a potential next PCB 😅

---

That's ok, half duplex is perfect 👌